roles/network_plugin/cilium/templates/cilium-config.yml.j2

```diff
@@ -145,3 +145,13 @@ data:
 
   native-routing-cidr: "{{ cilium_native_routing_cidr }}"
   auto-direct-node-routes: "{{ cilium_auto_direct_node_routes }}"
+
+  # Hubble settings
+{% if cilium_enable_hubble %}
+  enable-hubble: "true"
+  hubble-metrics: "{{ cilium_hubble_metrics }}"
```

Comment:
Is cilium_hubble_metrics is mandatory to be specified if cilium_enable_hubble is true?
If so, it is better to implement check cilium_hubble_metrics is specified in the case for users in roles/network_plugin/cilium/tasks/main.yml

Reply:
I just checked and an empty string works too. Cilium will not expose any metrics but this is a valid input.

Reply:
Thanks for your testing!